projects/js-packages/components/components/layout/container/index.jsx

```diff
@@ -48,6 +48,9 @@ Container.propTypes = {
 	horizontalSpacing: PropTypes.number,
 	/** Number of gap betwen rows, it gets multipled by 8px. Needs to be an integer */
 	horizontalGap: PropTypes.number,
+
+	/** Children to be inserted. */
+	children: PropTypes.node,
```

Comment:
needed to follow one
<img width="703" alt="image" src="https://user-images.githubusercontent.com/77539/166932578-969042f1-0ad7-4f11-bf04-6c71a1030bd9.png">